contracts/registry/tests/workspaces.rs

```diff
@@ -236,6 +247,8 @@ async fn migration_mainnet() -> anyhow::Result<()> {
 }
 
 #[ignore = "this test is not valid after the migration"]
```

Comment:
I think this test cannot be ingnored since we need the migration after the last changes